en/drafts/originals/text-mining-youtube-comments.md

```diff
@@ -424,7 +409,7 @@ print(paste("you created", "a dfm with", ndoc(dfmat_all), "documents and", nfeat
 
 #### Data Optimization
 
-In the next step, you will optimize the corpus to focus on the most meaningful words only. The following code removes words with fewer than four characters, as well as rare words (those that appear in less than 1% of documents, or that comprise less than .001% of the total corpus).
+In the next step, you will optimize the corpus to focus on the most meaningful words only. The following code removes words with fewer than four characters, as well as rare words (those that appear in less than 1% of documents, or that comprise less than 0.001% of the total corpus):
 
 ```
 dfmat_all <- dfm_keep(dfmat_all, min_nchar = 4)

```

Comment:
I noticed you mention 'less than 1% of documents above', but I see `min_docfreq = 0.01` in the code here below. Just checking this is correct?